content/docs/reference/providers/pulumi/aws/s3/bucket.md

```diff
@@ -0,0 +1,1827 @@
+---
+title: "Bucket"
+---
+
+<!-- WARNING: this file was generated by the Pulumi Terraform Bridge (tfgen) Tool. -->
+<!-- Do not edit by hand unless you're certain you know what you are doing! -->
+
+<style>
+  table td p { margin-top: 0; margin-bottom: 0; }
+</style>
+
+Provides a S3 bucket resource.
+
+> This content is derived from https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/r/s3_bucket.html.markdown.
+
+
+## Example Usage
```

Comment:
I wonder if we can collapse this entire section by default, with a little ▶️ collapser icon. I actually think it's right to include this at the top, however it tends to be lengthy and it can obscure the instructions on how to create new instances (which may be what most folks are wanting to see when they get here). Making it collapsed seems fine because if you want to see examples, you're going to see "Example Usage ▶️" and immediately know to click to expand.

The only hitch I see here is if you've linked to an anchor -- e.g. .../docs/reference/providers/pulumi/aws/s3/bucket/#private-bucket-w-tags -- if it's collapsed the right thing may not happen.

I actually think this is a small change that could make a huge positive usability impact, and lessen the feeling that our pages are so long and require so much scrolling to get to the meat.

Reply:
> The only hitch I see here is if you've linked to an anchor -- e.g. .../docs/reference/providers/pulumi/aws/s3/bucket/#private-bucket-w-tags -- if it's collapsed the right thing may not happen.

We could write some JavaScript that would make sure the right thing happens, e.g. if there current page's URL contains an anchor inside the collapsed example section, to automatically expand it.

Reply:
It does need to be collapsed and expandable, I agree. This was also on my list of things to try.